src/types/to_sql.rs

```diff
@@ -241,7 +231,7 @@ mod test {
     #[test]
     fn test_box() {
         let s: Box<str> = "Hello world!".into();
-        let r = s.to_sql();
+        let r = ToSql::to_sql(&s);
```

Comment:
I assume this is trying to beat deref coersion? I think it will still happen -- If there were `Box::to_sql` (I think) that would prevent it.

Also a comment that says that this is trying to defeat deref coersion.

Reply:
Actually maybe it would still not use deref coersion here... I'd like the comment either way though.